tests/setup.yaml

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Can I run these tests? If so, maybe a few details in the README would go a long way.